### PR TITLE
Fix code examples in the documentation

### PR DESCRIPTION
**Pull request by kroening**
To address issue #3566.

- [ ] Each commit message has a non-empty body, explaining why the change was made.
- n/a Methods or procedures I have added are documented, following the guidelines provided in CODING_STANDARD.md.
- n/a The feature or user visible behaviour I have added or modified has been documented in the User Guide in doc/cprover-manual/
- n/a Regression or unit tests are included, or existing tests cover the modified code (in this case I have detailed which ones those are in the commit message).
- n/a My commit message includes data points confirming performance improvements (if claimed).
- [ ] My PR is restricted to a single feature or bugfix.
- n/a White-space or formatting changes outside the feature-related changed lines are in commits of their own.

### CODE REVIEW

**Review by allredj — COMMENTED** [review]
:heavy_check_mark:
Passed Diffblue compatibility checks (cbmc commit: b01fc03b1e92eacafda35b74f7118f4fcf1879fb).
Build URL: https://travis-ci.com/diffblue/test-gen/builds/96399483

File: doc/cprover-manual/cbmc-tutorial.md [hunk 1]
```diff
@@ -241,7 +241,7 @@ void unlock()
 int main()
 {
   unsigned got_lock = 0;
-  int times;
+  int times = 10;
```
Comment (tautschnig) [inline doc/cprover-manual/cbmc-tutorial.md]: This makes the loop bounded, and thus the example not longer fits what the section is about.

**Review by tautschnig — CHANGES_REQUESTED** [review]

File: doc/cprover-manual/cbmc-tutorial.md (hunk 1 above)
Comment (kroening) [reply doc/cprover-manual/cbmc-tutorial.md]: indeed, fixed

**Review by allredj — COMMENTED** [review]
:heavy_check_mark:
Passed Diffblue compatibility checks (cbmc commit: d54d63b2a739ce808d716f91771da65d5d674ed4).
Build URL: https://travis-ci.com/diffblue/test-gen/builds/96402158

**Review by tautschnig — APPROVED** [review]

**Pull request merged**